CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix for momentary appearing of a lower zoom level tile labels during camera movement.  ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
 * Fix for location indicator not being rendered at the horizon when terrain is enabled. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
 * Fix for loading gltf models with interleaved buffers. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
+* Add mercator scale factor to 3D location puck, so that the 3D puck size won't increase as latitude increases. ([#1350](https://github.com/mapbox/mapbox-maps-android/pull/1350)
```

Comment:
```suggestion
* Add mercator scale factor to 3D location puck, so that the size of 3D puck won't increase as latitude increases. ([#1350](https://github.com/mapbox/mapbox-maps-android/pull/1350)
```